viz.yaml

```diff
@@ -411,13 +416,23 @@ publish:
     alttext:  >-
       High water use: Illinois (steel production), the Northeast (power generation), Arkansas (irrigation), & Gulf Coast (industry)
     relpath: "images/WU_stories_mobile_right.jpg"
+  -
+    id: placeholder
+    location: "images/placeholder.gif"
+    mimetype: image/gif
+    title: "Place Holder gif for lazy loading images"
+    alttext: "Place Holder gif for lazy loading images"
+    relpath: "images/placeholder.gif"
+    data-src: "images/WU_stories_static.jpg"
```

Comment:
so does this include the switch for desktop version vs mobile version of the image?

Reply:
It does not, that's a different functionality that I will work on after I create the new images

Reply:
The alt and title do not switch when the images swap.  So need the correct alt and title on the placeholder image